src/reformatters/dwd/icon_eu/forecast/dynamical_dataset.py

```diff
@@ -1,55 +1,151 @@
+        obstore_manager = ObstoreManager(dst_obstore=store, dst_root_path=dst_root_path)
+        coordinator = FtpTransferCoordinator(
+            ftp_manager=ftp_manager,
+            obstore_manager=obstore_manager,
         )
 
+        if nwp_init_hour == "all":
+            files_to_download = asyncio.run(
+                coordinator.calc_new_files_for_multiple_nwp_init_hours()
+            )
+        else:
+            init_hour = int(nwp_init_hour)
+            files_to_download = asyncio.run(
+                coordinator.calc_new_files_for_single_nwp_init_hour(init_hour)
+            )
+
+        src_ftp_paths: list[PurePosixPath] = []
+        dst_obstore_paths: list[str] = []
+        for transfer_job in files_to_download:
+            src_ftp_paths.append(transfer_job.src_ftp_path)
+            dst_obstore_paths.append(str(transfer_job.dst_obstore_path))
+
+        asyncio.run(
+            copy_files_from_ftp_to_obstore(
+                ftp_host=ftp_manager.ftp_host,
+                src_ftp_paths=src_ftp_paths,
+                dst_obstore_paths=dst_obstore_paths,
+                dst_store=store,
+            )
+        )
+
+        log.info(
+            "Finished downloading %d files from ftp://%s",
+            len(files_to_download),
+            ftp_manager.ftp_host,
+        )
+
+    def get_cli(self) -> typer.Typer:
+        """Create a CLI app with dataset commands."""
+        app = super().get_cli()
+        app.command()(self.archive_grib_files)
+        return app
+
     def validators(self) -> Sequence[validation.DataValidator]:
         """Return a sequence of DataValidators to run on this dataset."""
         # return (
         #     validation.check_analysis_current_data,
         #     validation.check_analysis_recent_nans,
         # )
-        raise NotImplementedError(
-            f"Implement `validators` on {self.__class__.__name__}"
+        return ()  # TODO(Jack): Return appropriate `validation.check_*`
+
+    def operational_kubernetes_resources(self, image_tag: str) -> Sequence[CronJob]:
+        """Return the kubernetes cron job definitions to operationally update
+        and validate this dataset."""
+        suspend = True  # Defaults to False, remove when you're ready to run operational updates and validation
+        ftp_to_obstore_cron_job = ArchiveGribFilesCronJob(
+            name=f"{self.dataset_id}-archive-grib-files",
+            # The files for each NWP init are ready 4 hours after the NWP init time. For example,
+            # the 00Z NWP init is ready at 4am. So let's run 4 times per day, starting at 4am.
+            schedule="0 4,10,16,22 * * *",
+            pod_active_deadline=timedelta(
+                minutes=120
+            ),  # TODO(Jack): Check how long it *actually* takes!
+            image=image_tag,
+            dataset_id=self.dataset_id,
+            cpu="14",
+            memory="30G",
+            shared_memory="12G",
+            ephemeral_storage="30G",
```

Comment:
since this will be doing very little cpu work lets try making this smaller. fine to increase if we need. we also don't need shared memory (we just use that for sharing an array w/ multiprocessing)
```suggestion
            cpu="3",
            memory="7G",
            ephemeral_storage="30G",
```